ignition/blueprint/ubuntu/bionic/ignition/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# This is an auto generated Dockerfile for ignition:ignition
+# generated from docker_images/create_ignition_image.Dockerfile.em
+FROM ubuntu:bionic
+
+# setup timezone
+RUN echo 'Etc/UTC' > /etc/timezone && \
+    ln -s /usr/share/zoneinfo/Etc/UTC /etc/localtime && \
+    apt-get update && apt-get install -q -y tzdata && rm -rf /var/lib/apt/lists/*
+
+# install packages
+RUN apt-get update && apt-get install -q -y \
+    dirmngr \
+    gnupg2 \
+    lsb-release \
+    && rm -rf /var/lib/apt/lists/*
+
+# setup keys
+RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys D2486D2DD83DB69272AFE98867170598AF249743
+
+# setup sources.list
+RUN . /etc/os-release \
+    && echo "deb http://packages.osrfoundation.org/gazebo/$ID-stable `lsb_release -sc` main" > /etc/apt/sources.list.d/gazebo-latest.list
+
+# install ignition packages
+RUN apt-get update && apt-get install -q -y \
+    ignition-blueprint=1.0.0-1* \
+    && rm -rf /var/lib/apt/lists/*
+
+# setup environment
+EXPOSE 11345
```

Comment:
In 20-20, I think me including this EXPOSE in gazebo image was a mistake. It was well intended to clarify what port was normally used for gazebo server, but if the user was using this image in a software defined network, or was using multiple containers, or with non-default gazebo ports, then this really gets in the way as it can not be unset. There's been a feature request in docker from 2014, but I don't think that status has changed. 
https://github.com/moby/moby/issues/3465

I'd argue to omit this from the ignition template, and let downstream users define what or what not to expose.

Reply:
Oh good, thanks for pointing it out! Removed from the templates in https://github.com/osrf/docker_templates/pull/84/commits/8534eeeb0906a9f27f6ca5715a0a53732cabb571